scripts/test-reviewer.mjs

```diff
@@ -231,269 +231,242 @@ fse.readdirSync(testsBuildDirectory).forEach(function (directory) {
       }
     });
 
-    collectedTestsData
-      .sort((a, b) => {
```

Comment:
For reviewers: because of the formatting after removing the offending sort in, the actual change has been obscured quite a bit for `test-reviewer.mjs`. The following has been removed:

<details>
<summary>Removed the invalid sort function</summary>

```javascript
    .sort((a, b) => {
        const commandsInfoA = a.commandsInfo;
        const commandsInfoB = b.commandsInfo;

        // The whole number will always be the same in the collection
        function extractPresentationNumber(data) {
          let presentationNumber;

          // Loop through the keys in testData
          for (const key in data) {
            // Check if the key has data
            if (data[key].length > 0) {
              // Extract presentation number from the first item in the array
              presentationNumber = parseInt(data[key][0].presentationNumber);
            }
          }

          return presentationNumber;
        }

        const wholeNumberA = extractPresentationNumber(commandsInfoA);
        const wholeNumberB = extractPresentationNumber(commandsInfoB);

        return wholeNumberA - wholeNumberB;
      })
```
</details>